app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
     pass
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_list = []
+
+    for human in people:
+        new_human = Person(human["name"], human["age"])
+        new_list.append(new_human)
+
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife =\
+                Person.people[person["wife"]]
```

Comment:
I don't recommend you use `\` for writing in a new line. For example, you can make `Person.people[person["name"]]` as a variable